azure-pipelines/run_test_steps.yml

```
@@ -45,7 +47,7 @@ steps:
    python -c "import os; [ print(e,v) for (e,v) in os.environ.items() ]"
   displayName: 'Get Python Information'
 
-- script: 'python -m pip install -U pip setuptools tox'
+- script: 'python -m pip install -U pip setuptools tox coveralls pyyaml'
```

Wasn't the dropping of `.coveralls.yml` so that we didn't need `pyyaml`?  https://github.com/twisted/twisted/pull/1574/files#r602889624  In case that is so...

```suggestion
- script: 'python -m pip install -U pip setuptools tox coveralls'
```